tests/integration/steps/integration_steps.go

```diff
@@ -0,0 +1,622 @@
+
+	splitOptions := strings.Split(options, "-")
+	medusaEnabled := "true"
+	reaperEnabled := "true"
+	monitoringEnabled := "true"
+
+	if Find(splitOptions, "nomedusa") {
+		medusaEnabled = "false"
+	}
+
+	if Find(splitOptions, "noreaper") {
+		reaperEnabled = "false"
+	}
+
+	if Find(splitOptions, "nomonitoring") {
+		monitoringEnabled = "false"
+	}
+
+	newGenSize, _ := strconv.Atoi(strings.ReplaceAll(strings.ReplaceAll(cassandraHeap, "M", ""), "G", ""))
+	helmValues := map[string]string{
+		"cassandra.heap.size":           cassandraHeap,
+		"cassandra.heap.newGenSize":     strconv.Itoa(newGenSize/2) + "M",
+		"stargate.heapMB":               strings.ReplaceAll(stargateHeap, "M", ""),
+		"medusa.enabled":                medusaEnabled,
+		"reaper.enabled":                reaperEnabled,
+		"reaper-operator.enabled":       reaperEnabled,
+		"kube-prometheus-stack.enabled": monitoringEnabled,
+	}
+	deployCluster(t, customValues, helmValues)
+}
+
+func CheckK8ssandraNamespaceWasCreated(t *testing.T) {
+	t.Logf("I can see the %s namespace in the list of namespaces", namespace)
+
+	kubectlOptions := k8s.NewKubectlOptions("", "", "default")
+	_, err := k8s.GetNamespaceE(t, kubectlOptions, namespace)
+	g(t).Expect(err).To(gomega.BeNil())
+}
+
+func CheckSecretIsPresent(t *testing.T, secret string) {
+	kubectlOptions := k8s.NewKubectlOptions("", "", namespace)
+	_, err := k8s.GetSecretE(t, kubectlOptions, secret)
+	g(t).Expect(err).To(gomega.BeNil())
+}
+
+func CheckNamespaceIsAbsent(t *testing.T) {
+	g(t).Eventually(func() bool {
+		return namespaceIsAbsentOrTerminating(t, namespace)
+	}, RETRY_TIMEOUT, RETRY_INTERVAL).Should(gomega.BeTrue())
+}
+
+func namespaceIsAbsentOrTerminating(t *testing.T, ns string) bool {
+	kubectlOptions := k8s.NewKubectlOptions("", "", "default")
+	namespaceObject, _ := k8s.GetNamespaceE(t, kubectlOptions, namespace)
+	if namespaceObject == nil {
+		// namespace was deleted
+		return true
+	}
+
+	if namespaceObject.Status.Phase == v1.NamespaceTerminating {
+		// namespace is terminating, which is good enough
+		return true
+	}
+	return false
+}
+
+func CreateNamespace(t *testing.T) {
+	namespace = fmt.Sprintf("k8ssandra%s", time.Now().Format("2006010215040507"))
+	log.Println(fmt.Sprintf("Creating namespace %s", namespace))
+	kubectlOptions := k8s.NewKubectlOptions("", "", "default")
+	k8s.CreateNamespace(t, kubectlOptions, namespace)
+}
+
+func UninstallK8ssandraHelmRelease(t *testing.T) {
+	err := RunShellCommand(exec.Command("helm", "uninstall", releaseName, "-n", namespace))
+	g(t).Expect(err).To(gomega.BeNil(), "Failed uninstalling K8ssandra Helm release")
+}
+
+func UninstallTraefikHelmRelease(t *testing.T) {
+	err := RunShellCommand(exec.Command("helm", "uninstall", "traefik", "-n", "traefik"))
+	g(t).Expect(err).To(gomega.BeNil(), "Failed uninstalling Traefik Helm release")
+}
+
+func DeleteK8ssandraNamespace(t *testing.T) {
+	deleteNamespace(t, namespace)
+}
+
+func DeleteTraefikNamespace(t *testing.T) {
```

Comment:
To which packages are these functions meant to be exported to?

Reply:
They are used by [cleanupCluster](https://github.com/k8ssandra/k8ssandra/blob/5c018349a3fd6ee291a419fc082dfb85d04bdaee/tests/integration/integration_suite_test.go#L27) in the `integration` package.